libs/coin-modules/coin-ton/src/bridge/bridgeHelpers/api.ts

```diff
@@ -82,6 +86,37 @@ export async function fetchAccountInfo(addr: string): Promise<TonAccountInfo> {
   };
 }
 
+export async function fetchJettonTransactions(
+  addr: string,
+  opts?: {
+    jettonMaster?: string;
+    startLt?: string;
+    endLt?: string;
+  },
+): Promise<TonJettonTransfer[]> {
+  const address = Address.parse(addr);
+  const urlAddr = address.toString({ bounceable: false, urlSafe: true });
+  let url = `/jetton/transfers?address=${urlAddr}&limit=256`;
+  if (opts?.jettonMaster != null) url += `&jetton_master=${opts.jettonMaster}`;
```

Comment:
If I understand well, the jettonMaster should never be null. Isn't it?

Reply:
Hi! When synchronizing the account, we don't search by token, we search all of the transfers and then we separate them by token with the function [getSubAccounts](https://github.com/LedgerHQ/ledger-live/blob/f063790970e66efb7db488bfef53b8df5b99ea7f/libs/coin-modules/coin-ton/src/synchronisation.ts#L147)